di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/handlers/IpvHandler.java

```diff
@@ -188,6 +188,15 @@ public JSONObject getUserInfo(AccessToken accessToken) {
 
         HTTPResponse userInfoHttpResponse = sendHttpRequest(userInfoRequest.toHTTPRequest());
 
+        int statusCode = userInfoHttpResponse.getStatusCode();
+        if (statusCode != HTTPResponse.SC_OK) {
+            logger.error(
+                    "User info request failed with status code {}: {} ",
+                    statusCode,
+                    userInfoHttpResponse.getContent());
+            throw new RuntimeException("User info request failed with status code " + statusCode);
```

Comment:
I'd be inclined to add the more useful info from the error description to the runtime exception - the status code is very generic